lib/router/router.js

```diff
@@ -109,6 +126,17 @@ p.getInfo = function(req) {
     }
   }
 
+  // forward original image if no operation on image and browser support image format
+  if (routeInfo.imageSteps.length === 0 && (this.options.noop || routeInfo.urlInfo.query.noop)) {
```

Comment:
No, if you specify `useOriginal`, it will not generate an optimizedOriginal image -- it will skip that step. The only processing of sharp if there are no other steps, is to change the format to the optimal format (webp for supported browsers, otherwise jpeg). Of course you can still request a specific format. If you do not want the format changed, you would new a need a new option like `autoFormat:false`.